specify/specify7
# Add the possibibility to collapse conditional formatters

Fixes #4370 

### Checklist

      and self-explanatory (or properly documented)

### Testing instructions

- Open the formatter UI
- Verify that checking conditional formatter field box displays a new UI for conditionals
- Verify that unchecking conditional formatter displays only the separator + field component 
- test adding new field 
- test deleting field 
- test adding a conditional value
- test deleting conditional with icon button
- test deleting conditional with Delete Definition button
- test collapsible function when conditional checked
